package.json

```diff
@@ -35,7 +32,6 @@
     "tool-bar": {
       "description": "A container for buttons at workspace edge",
       "versions": {
-        "0.1.0": "provideToolBarLegacy",
         "1.0.0": "provideToolBar"
```

Comment:
Not necessarily, in my [research](https://github.com/suda/tool-bar/issues/141) only 3 package used the extended API. Nuclide is up-to-date, Flex Tool Bar hopefully soon, and don't really care about Atom-macros.